src/johari/Johari.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Johari from './Johari';
+
+describe('Johari', () => {
```

Comment:
@lucyconklin Maybe we could also do a mount test to make sure it contains `.AdjectiveList`?